backend/api/game.go

```diff
@@ -3,55 +3,17 @@ package api
 import (
 	"net/http"
 
-	"github.com/google/uuid"
 	"github.com/jak103/usu-gdsf/db"
 	"github.com/jak103/usu-gdsf/log"
 	"github.com/labstack/echo/v4"
 )
 
-func getGameByID(c echo.Context) error {
-	db, err := db.NewDatabaseFromEnv()
-
-	if err != nil {
-		log.WithError(err).Error("Unable to use database")
-		return err
-	}
-	gameID, _ := uuid.Parse(c.Param("ID"))
-
-	if result, err := db.GetGameByID(gameID); err != nil {
-		log.Error("An error occurred while getting game records: %v", err)
-		return err
-	} else {
-		return c.JSON(http.StatusOK, []interface{}{result})
-	}
-
-}
-
-// for downloaded games, this route allows you to play them
-func gameDownload(c echo.Context) error {
-	return c.JSON(http.StatusOK, "Download a game")
+func game(c echo.Context) error {
+	return c.JSON(http.StatusOK, "Successful game get!")
 }
 
-func getGames(c echo.Context) error {
+func getAllGames(c echo.Context) error {
 	db, err := db.NewDatabaseFromEnv()
-	query := c.Request().URL.Query()
```

Comment:
Looks like a merge conflict happened here and I think we need this data instead.